client/core/core.go

```diff
@@ -1176,7 +1175,7 @@ func (c *Core) verifyRegistrationFee(wallet *xcWallet, dc *dexConnection, coinID
 	trigger := func() (bool, error) {
 		confs, err := wallet.Confirmations(coinID)
 		if err != nil {
-			return false, fmt.Errorf("Error getting confirmations for %s: %v", hex.EncodeToString(coinID), err)
+			return false, fmt.Errorf("Error getting confirmations for %s: %v", coinIDString(assetID, coinID), err)
```

Comment:
I think that `wallet.AssetID` is the relevant asset ID you can use here. Not sure why it was ever an input to `verifyRegistrationFee` now that I see that.  Perhaps it was just missed.

Reply:
Replaced all instances of regFeeAssetID with wallet.AssetID.

Reply:
@itswisdomagain Touches the code you were just in.  Can you have a peek and see if this makes sense.  I'm confused why asset id was ever an input argument.

Reply:
Yeah, makes good sense. I like the usage of `wallet.AssetID` instead of `regFeeAssetID` or an `assetID` parameter.